jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb

```diff
@@ -0,0 +1,688 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%mavenRepo snapshots https://oss.sonatype.org/content/repositories/snapshots/\n",
+    "\n",
+    "%maven ai.djl:api:0.7.0-SNAPSHOT\n",
+    "%maven ai.djl:model-zoo:0.7.0-SNAPSHOT\n",
+    "%maven ai.djl:basicdataset:0.7.0-SNAPSHOT\n",
+    "%maven org.slf4j:slf4j-api:1.7.26\n",
+    "%maven org.slf4j:slf4j-simple:1.7.26\n",
+    "%maven ai.djl.mxnet:mxnet-engine:0.7.0-SNAPSHOT\n",
+    "%maven ai.djl.mxnet:mxnet-native-auto:1.7.0-a"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%loadFromPOM\n",
+    "<dependency>\n",
+    "    <groupId>tech.tablesaw</groupId>\n",
+    "    <artifactId>tablesaw-jsplot</artifactId>\n",
+    "    <version>0.30.4</version>\n",
+    "</dependency>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load ../utils/plot-utils.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ai.djl.Model;\n",
+    "import ai.djl.basicdataset.FashionMnist;\n",
+    "import ai.djl.metric.Metrics;\n",
+    "import ai.djl.ndarray.NDArray;\n",
+    "import ai.djl.ndarray.NDManager;\n",
+    "import ai.djl.ndarray.types.DataType;\n",
+    "import ai.djl.ndarray.types.Shape;\n",
+    "import ai.djl.nn.Activation;\n",
+    "import ai.djl.nn.Blocks;\n",
+    "import ai.djl.nn.SequentialBlock;\n",
+    "import ai.djl.nn.convolutional.Conv2D;\n",
+    "import ai.djl.nn.core.Linear;\n",
+    "import ai.djl.nn.pooling.Pool;\n",
+    "import ai.djl.training.DefaultTrainingConfig;\n",
+    "import ai.djl.training.EasyTrain;\n",
+    "import ai.djl.training.Trainer;\n",
+    "import ai.djl.training.dataset.Dataset;\n",
+    "import ai.djl.training.dataset.ArrayDataset;\n",
+    "import ai.djl.training.evaluator.Accuracy;\n",
+    "import ai.djl.training.listener.TrainingListener;\n",
+    "import ai.djl.training.loss.Loss;\n",
+    "import ai.djl.training.optimizer.Optimizer;\n",
+    "import ai.djl.training.optimizer.learningrate.LearningRateTracker;\n",
+    "import tech.tablesaw.api.*;\n",
+    "import tech.tablesaw.plotly.api.*;\n",
+    "import tech.tablesaw.plotly.components.*;\n",
+    "import tech.tablesaw.plotly.Plot;\n",
+    "import tech.tablesaw.plotly.components.Figure;\n",
+    "import org.apache.commons.lang3.ArrayUtils;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "attributes": {
+     "classes": [],
+     "id": "",
+     "n": "1"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "NDManager manager = NDManager.newBaseManager();\n",
+    "\n",
+    "SequentialBlock block = new SequentialBlock();\n",
+    "\n",
+    "block\n",
+    "                .add(Conv2D.builder()\n",
```

Comment:
Please also keep comments http://d2l.ai/chapter_convolutional-modern/alexnet.html#capacity-control-and-preprocessing